specs/sharding/beacon-chain.md

```diff
@@ -186,26 +193,35 @@ class DataCommitment(Container):
     length: uint64
 ```
 
-### `ShardHeader`
+### `ShardBlobBodySummary`
 
 ```python
-class ShardHeader(Container):
-    # Slot and shard that this header is intended for
-    slot: Slot
-    shard: Shard
+class ShardBlobBodySummary(Container):
```

Comment:
Maybe DataSummary?

Reply:
I want to make clear it's another variant of the same body type, although you are right that it's the data that is really getting summarized into a `Root`. Let's add this to the other naming changes for a later PR.